backend/src/main/kotlin/blue/mild/covid/vaxx/error/ExceptionHandling.kt

```diff
@@ -43,7 +45,10 @@ fun Application.installExceptionHandling() {
 
         exception<Exception> { cause ->
             logger.error(cause) { "Exception occurred in the application: ${cause.message}" }
-            call.errorResponse(HttpStatusCode.InternalServerError, cause.message)
+            call.errorResponse(
+                HttpStatusCode.InternalServerError,
+                "Server was unable to fulfill the request, please contact administrator with request ID: ${MDC.get(CALL_ID)}"
```

Comment:
A jsme shopny sparovat tu chuby pres to CALL_ID? Pokud jo, tak cajk.

Reply:
dokonce se vsema logama, co mame pro dany call, propisujeme tohle call id do kazdeho radku logu

```
2021-03-07 15:14:05,296[E][remoteHost:localhost][callId:273d86cd-c11c-2634-b696-b14d214dc906] ExceptionHandler - Exception occurred in the application: Fuck
2021-03-07 15:14:05,332[D][remoteHost:localhost][callId:273d86cd-c11c-2634-b696-b14d214dc906] HttpCallLogger - 500 Internal Server Error: GET - /api/insurance-company
```